main/guides/getting-started/README.md

```diff
@@ -1,163 +1,199 @@
 :::
 
-## Install Go
+We support running a local Agoric blockchain with [docker-compose](https://docs.docker.com/compose/).
 
-Download Go from [go.dev/doc/install](https://go.dev/doc/install) and follow the instructions for your platform.
+Our user interfaces integrate with the [keplr](https://www.keplr.app/) wallet.
 
-```shell
-go version # Version 1.20.3 or higher
+## Create a project with the basic starter kit
+
+Choose a name such as `demo` and then:
+
+```sh
+yarn create @agoric/dapp demo
+cd demo
 ```
 
-## Install Node.js
+::: tip TODO: make dapp-game-places the default
 
-Download Node.js from [Node.js](https://nodejs.org/) and follow the instructions for your platform.
-We recommend installing the LTS version of node 18.
+Until `dapp-game-places` is the default, use `--dapp-base` etc.
+to refer to it:
+
+```sh
+yarn create @agoric/dapp \
+  --dapp-base https://github.com/agoric-labs/ \
+  --dapp-template dapp-game-places \
+  demo
 
-```shell
-node --version # LTS version such as 18.16.0
 ```
 
-**Note:** Agoric will support all long-term support (LTS) versions of Node.js. 
+:::
 
-## Install the Yarn Package Manager
+### Project structure
 
-Follow [Yarn Installation](https://classic.yarnpkg.com/en/docs/install)
-instructions. For example:
+The `demo` directory now has:
 
-```shell
-npm install --global yarn
-yarn --version # 1.22.10 or higher
-```
+- **README.md**
+- **package.json** as usual for a JavaScript project, along with **yarn.lock**
+- **contract/** - smart contract source code, tests, etc.
+- **ui/** - web UI
+- **docker-compose.yml** - for running a local Agoric blockchain
 
-## Install Git
+::: tip TODO: prune api, agstate
 
-Follow [Git installation instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) or use your platform's package manager.
+Disregard these contents, for now:
 
-```shell
-npm install --global git
-git --version # 2.25.0 or higher
+- **\_agstate**
+- **api**
+  :::
+
+      node_modules
+
+## Install dependencies
+
+The UI depends on the React framework, and the contract depends on
+the Agoric framework. The packages in this project also have
+development dependencies for testing, code formatting, and static analysis.
+
+```sh
+yarn install  # may take several minutes
 ```
 
-## Clone the Agoric SDK
+::: tip Troubleshooting `yarn install`
 
-```shell
-git clone --branch community-dev https://github.com/Agoric/agoric-sdk
-cd agoric-sdk
+If you run into errors during install or build, make sure you have the relevant developer tools installed. For example, on Debian or Ubuntu Linux, you can run `sudo apt get install build-essential` to install these tools.
+
+Also, check that you are on a
+[supported platform](#platform-linux-shell-or-equivalent) and
+not native Windows.
+:::
+
+## Start a local Agoric blockchain
+
+To run a local Agoric blockchain with [docker-compose](https://docs.docker.com/compose/):
+
+```sh
+yarn start:docker
```

Comment:
Need to add `cd contract` for this command to work.
Also, can we change this to be `yarn docker:start` to match with other docker related commands?

Reply:
yeah... I'm not sure whether to `cd` or to tweak the top-level `package.json` so that it's not necessary.

re `start:docker` vs `docker:start` ... I got `start:docker` from the indexing workshop a while back. I don't know how widespread the convention is. Something to think about, yeah.

Reply:
> I'm not sure whether to cd or to tweak the top-level package.json so that it's not necessary.

I did the tweak:

2023-12-04 17:38 c46b334 build: make docker scripts available at top level

still considering `start:docker` vs `docker:start`.